blockchain.go

```diff
@@ -441,6 +440,7 @@ func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPubli
 	options := make([]fvm.BootstrapProcedureOption, 0)
 	options = append(options,
 		fvm.WithInitialTokenSupply(supply),
+		fvm.WithRestrictedAccountCreationEnabled(false),
```

Comment:
Tests fail with `missing account created event` with only this change.

Reply:
I can take a look. I know what the problem is!